src/Components/Artwork/GridItem.tsx

```diff
@@ -156,7 +160,7 @@ export const ArtworkGridItem = styled(ArtworkGridItemContainer)`
   }
 `
 
-export default createFragmentContainer(ArtworkGridItem, {
+export default createFragmentContainer(withSystemContext(ArtworkGridItem), {
```

Comment:
This is the one significant concern I have with this PR. This _seems_ like it should be just fine, but `GridItem` was previously relying on the `user` prop being passed into it, and `ArtworkGrid` (the only thing that renders a `GridItem`) was _also_ relying on the `user` prop being passed into _it_. That seems kind of strange to me, since `GridItem` could have just subscribed to the context to get the `user`. It makes me wonder if there is a reason that it's relying on it being passed in, and if this change will break something in a very unexpected way.

Reply:
The grid stuff is like pre v1 of Reaction, right when we starting evaluating our new tech stack and were figuring things out. Tech debt.